app/generator/content/part2.js

```diff
@@ -0,0 +1,120 @@
+const part2 = (statement) => {
+  const reductions = mapReductions(statement.funding)
+
+  return [
+    { text: 'Part 2. Calculation', style: 'header2' },
+    { text: `We calculated the total ${statement.scheme.shortName} payment amount by adding together payments for all the standards which are part of your agreement, as shown in this table.` },
+    getTable(statement.funding, reductions),
+    {
+      text: [
+        'You can check which land parcels are part of each standard in the Rural Payments service at ',
+        { text: 'www.gov.uk/claim-rural-payments', link: 'https://www.gov.uk/claim-rural-payments', style: 'link' },
+        '.'
+      ]
+    },
+    ...getReductions(reductions)
+  ]
+}
+
+const mapReductions = (funding) => {
```

Comment:
do you think this and the funcs below need unit tested?

appreciate you have unit tests for the `part2` func which calls out to these but it deals more with arranging how the outputs of these are displayed, rather than how they are created

Reply:
Yes I think that's a fair comment.  I was considering breaking down part 2 into a more full module rather than a single file so this comment has persuaded me this is the right thing to do.  

Some of these low level functions have a decent degree of complexity so it does make sense to have more targeted tests against them to cover any edge cases.  For the table itself not so much (I've added some anyway) but for the others it would be extra assurance.  Particularly when we start mapping other schemes to the table.